src/libraries/System.Runtime/tests/System/DecimalTests.cs

```diff
@@ -224,11 +224,48 @@ public void Ctor_Double(double value, int[] bits)
         [InlineData(double.MinValue)]
         [InlineData(double.PositiveInfinity)]
         [InlineData(double.NegativeInfinity)]
+        [InlineData(79228162514264337593543950335.0)]
```

Comment:
This number is decimal.MaxValue, which is actually stored as 7922816251426433759354395033**6**.0 as a double, which has an exponent of 96 and should overflow when cast to a decimal.